src/components/SplitPane.tsx

```diff
@@ -108,12 +115,14 @@ export function SplitPane(props: SplitPaneProps) {
     sideSeparation = 'start',
     initialSeparation = '50%',
     initialClosed = false,
+    minimalSize,
     onChange = () => null,
     children,
   } = props;
 
   const [isSidePaneClosed, toggle] = useToggle(initialClosed);
   const parentRef = useRef<HTMLDivElement>(null);
+  const touchedRef = useRef<boolean>(false);
```

Comment:
You could get rid of this ref by allowing your `isSidePaneClosed` to take a third initial value (`null` for example)

Reply:
I think this ref is OK. The "isTouched" information is not needed outside the component and shouldn't trigger a rerender when it changes.